digital_asset_types/src/dao/scopes/asset.rs

```diff
@@ -282,16 +310,25 @@ pub async fn get_by_id(
     let (asset, data) = asset_data;
     let authorities: Vec<asset_authority::Model> = asset_authority::Entity::find()
         .filter(asset_authority::Column::AssetId.eq(asset.id.clone()))
+        .order_by_asc(asset_authority::Column::AssetId)
         .all(conn)
         .await?;
     let creators: Vec<asset_creators::Model> = asset_creators::Entity::find()
         .filter(asset_creators::Column::AssetId.eq(asset.id.clone()))
-        .order_by(asset_creators::Column::Position, Order::Asc)
+        .order_by_asc(asset_creators::Column::Position)
         .all(conn)
         .await?;
     let grouping: Vec<asset_grouping::Model> = asset_grouping::Entity::find()
         .filter(asset_grouping::Column::AssetId.eq(asset.id.clone()))
-        .filter(asset_grouping::Column::Verified.eq(true))
+        .filter(asset_grouping::Column::GroupValue.is_not_null())
+        .filter(
+            Condition::any()
+                .add(asset_grouping::Column::Verified.eq(true))
+                // Older versions of the indexer did not have the verified flag. A group would be present if and only if it was verified.
```

Comment:
Yes. You could chose to default null to true during the migration, but that will be trickier to do at runtime because its a very slow operation if you index all NFTs. We ran into some pretty nasty states as a result. This is safer / easier.